.github/workflows/validate-migration.yaml

```diff
@@ -0,0 +1,101 @@
+name: validate-migration
+
+on: [ push ]
+
+env:
+  # Constants
+  P4_DOWNLOAD_URL: "https://www.perforce.com/downloads/perforce/r23.1/bin.linux26x86_64/p4api-glibc2.3-openssl1.0.2.tgz"
```

Comment:
Locally we currently use the 22 version, but since the tests pass I’m inclined to say let’s just upgrade?

Reply:
Ah, my bad. You're comfortable keeping this change though?

Reply:
Yeah, I think there's no way we can be more confident than with this CI pipeline so what else are we gonna do to validate it :)